### PR TITLE
Fsh/event type labels

### LINKED ISSUE #22: Add offseason label to offseason events

### PR DESCRIPTION
**Pull request by fharding1**
Fixes #22

### CODE REVIEW

File: src/routes/home/index.tsx [hunk 1]
```diff
@@ -64,7 +65,14 @@ export default () => (
                     <li key={e.key}>
                       <a href={`/events/${e.key}`}>
                         {e.shortName}
-                        <DateDisplay date={e.parsedDate} />
+                        <div class={infoClass}>
+                          <p>
+                            {screen.width <= 768
```
Comment (calebeby) [inline src/routes/home/index.tsx]: Nope. Use CSS media queries, not JS width checks. This does not work when you resize it

File: src/utils.tsx [hunk 2]
```diff
@@ -69,6 +69,27 @@ const toPercentage = (val: number) => Math.round(val * 100) + '%'
 
 const toPrettyNumber = (val: number) => Math.round(val * 10) / 10
 
+const eventTypeNames = new Map<number, string>([
+  [0, 'Regional'],
```
Comment (calebeby) [inline src/utils.tsx]: Hm, can't tell why these are wrong. Not sure if it is TBA's fault or ours.

Michigan subdivisions should be district championship division, but they are championship finals.
Michigan should be district championship, but it is district championship division.
Pacific Northwest should be district championship, but it is district championship division.

File: src/routes/home/index.tsx (hunk 1 above)
Comment (fharding1) [reply src/routes/home/index.tsx]: How do you use media queries to abbreviate? I guess we could have two elems and only display one or another...

Comment (calebeby) [reply src/routes/home/index.tsx]: Yeah, I think that is the best solution

File: src/utils.tsx [hunk 3]
```diff
@@ -69,15 +76,39 @@ const toPercentage = (val: number) => Math.round(val * 100) + '%'
 
 const toPrettyNumber = (val: number) => Math.round(val * 10) / 10
 
+const eventTypeNames = new Map<number, string>([
+  [0, ''],
+  [1, ''],
+  [5, 'DCPD'],
+  [2, 'DCP'],
```
Comment (calebeby) [inline src/utils.tsx]: DCMP please for both of these

File: src/utils.tsx [hunk 4]
```diff
@@ -69,15 +76,39 @@ const toPercentage = (val: number) => Math.round(val * 100) + '%'
...
+  [3, 'CPD'],
+  [4, 'CPF'],
+  [6, ''],
+  [99, 'Off'],
+  [100, 'Pre'],
+  [-1, '--']
```
Comment (calebeby) [inline src/utils.tsx]: I think let's not have a label for this

File: src/utils.tsx [hunk 5]
```diff
@@ -69,15 +76,39 @@ const toPercentage = (val: number) => Math.round(val * 100) + '%'
 
 const toPrettyNumber = (val: number) => Math.round(val * 10) / 10
 
+const eventTypeNames = new Map<number, string>([
+  [0, ''],
+  [1, ''],
+  [5, 'DCPD'],
+  [2, 'DCP'],
+  [3, 'CPD'],
+  [4, 'CPF'],
```
Comment (calebeby) [inline src/utils.tsx]: We don't need labels for championship. If you really want labels, can it be CMP for both?

File: src/routes/home/index.tsx [hunk 6]
```diff
@@ -3,14 +3,16 @@ import { home } from './style.sss'
...
+import { sortEvents, hasValidJWT, eventTypeName, abbreviate } from '../../utils'
 import Spinner from '../../components/spinner'
 import List from '../../components/list'
 import DateDisplay from '../../components/date-display'
 import FRCEvent from '../../models/frc-event'
 import Button from '../../components/button'
 import { route } from 'preact-router'
 import Header from '../../components/header'
+import { info as infoClass } from './style.sss'
+import { dcmp, cmp, off, pre } from './style.sss'
```
Comment (calebeby) [inline src/routes/home/index.tsx]: Can these be on the same line?

**Comment by fharding1** [comment]
Sure I guess but I think it's neater with them separated

File: src/components/icon/style.sss [hunk 7]
```diff
@@ -1,5 +1,5 @@
 .icon
-  width: 24px
-  height: 24px
+  width: 16px
```
Comment (calebeby) [inline src/components/icon/style.sss]: Why?

Comment (fharding1) [reply src/components/icon/style.sss]: It was too big

**Comment by calebeby** [comment]
In the header?

**Comment by fharding1** [comment]
Aren't we using it for the calendar too?

**Comment by calebeby** [comment]
Yes we are but changing the size here causes side effects throughout the
app. Can you change it directly in date?

**Pull request merged**